kernel/data-pipeline/dialect/opengauss/src/main/java/org/apache/shardingsphere/data/pipeline/opengauss/ingest/OpenGaussWALDumper.java

```diff
@@ -96,7 +94,11 @@ protected void runBlocking() {
                 int times = reconnectTimes.incrementAndGet();
                 log.error("Connect failed, reconnect times={}", times, ex);
                 if (isRunning()) {
-                    Thread.sleep(5000L);
+                    try {
+                        Thread.sleep(5000L);
+                    } catch (final InterruptedException ignored) {
+                        break;
+                    }
```

Comment:
If InterruptedException is ignored, then CDCJob might consider this dumper running fine without error and trigger next step.